src/microbe_stage/Microbe.cs

```diff
@@ -2108,6 +2108,75 @@ private Transform GetNewPhysicsRotation(Transform transform)
         return new Transform(new Basis(slerped), transform.origin);
     }
 
+    /// <summary>
+    ///   Adds torque to reach the targeted rotation
+    /// </summary>
+    private void ApplyPhysicsRotation(PhysicsDirectBodyState physicsState)
+    {
+        var transform = physicsState.Transform;
+        var target = transform.LookingAt(LookAtPoint, new Vector3(0, 1, 0));
+        if (transform == target)
+            return;
+
+        // For now strength uses only base strength
+        // Can later be affected by cilia
+        var strength = BaseRotationalStrength();
+
+        // Calculate the angle to reach the target rotation in degrees
+        var angleToTarget = target.basis.GetEuler().y - transform.basis.GetEuler().y;
+        angleToTarget = (float)(angleToTarget * (180.0f / Math.PI));
+        if (angleToTarget > 180.0f)
+        {
+            angleToTarget = -360.0f + angleToTarget;
+        }
+        else if (angleToTarget < -180.0f)
+        {
+            angleToTarget = 360.0f + angleToTarget;
+        }
+
+        // Maximum desired angular velocity
+        // If the value is to high the microbe will not brake in time
+        // If it is to low the microbe is rotating slower than it could
+        var maxVelocity = 1.0f + (strength * 0.3f) / (HexCount * 40.0f);
+
+        // Angular velocity of the microbe in a range from -90(-maxVelocity) to 90(maxVelocity)
+        var velocityTo90 = physicsState.AngularVelocity.y / maxVelocity * 90.0f;
```

Comment:
I have trouble understanding what is going on here. What are, in the end, the units of the velocity to 90? 
The name makes me think of degrees/s, but then the formula seems more of a proportion (unitless) times degrees...